CI/before_script.linux.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh -e
+#!/bin/bash -ex
```

Comment:
did this need to switch to bash specifically? Isn't it better to be agnostic?

Reply:
dash (default sh in ubuntu) doesn't support =~ operator that I needed when testing for clang, so bash it is.

However, if you can do the same and set it bash sh, please send a PR :)

Don't forget, this is for CI builds on Travis-CI and AppVeyor where bash is expected. Linux based releases are not built with these scripts, that is the job of the package managers.